setup.py

```diff
@@ -40,6 +40,7 @@
         # Installing sasl on Windows is rather painful, so use the pure python
         # implementation on Windows
         'pure-sasl>=0.3.0' if WINDOWS else 'sasl>=0.2.1',
+        'six==1.13.0'
```

Comment:
minor: why pin to this particular verion of six? 

we only use `ensure_binary` from six, and it was added in an earlier version of six and hasn't changed since then (and unlikely will change in a future version )

thank you!

Reply:
Thanks for the comment. As a general practice, I think of it as a best practice to always pin dependency versions. Several times, I've seen environments break when an underlying dependency gets inadvertently updated. I've also released a package update myself that accidentally broke other people environments. So it's my preference to only update a package when you explicitly want to do so, rather than simply always pulling the latest version by default.

Reply:
@dknupp that makes sense. 

From another prospective, we have a conda environments with hundreds of conda packages, and I've seen when version pinning sometimes creates conflicting / unresolvable dependencies. Notice `sasl` has `>= 0.2.1` and I can imagine changes in `sasl` can break `thift-sasl` more easily than `six`.

https://conda-forge.org/docs/maintainer/pinning_deps.html

> Especially widely used libraries with restricted version compatibilities increase the danger of fractioning the package space